lib/flipper/adapters/cache_store.rb

```diff
@@ -0,0 +1,104 @@
+module Flipper
+  module Adapters
+    # Public: Adapter that wraps another adapter with the ability to cache
+    # adapter calls in ActiveSupport::CacheStore caches.
+    #
+    class CacheStore
+      include ::Flipper::Adapter
+
+      Version = 'v1'.freeze
+      Namespace = "flipper/#{Version}".freeze
+      FeaturesKey = "#{Namespace}/features".freeze
+
+      # Private
+      def self.key_for(key)
+        "#{Namespace}/feature/#{key}"
+      end
+
+      # Internal
+      attr_reader :cache
+
+      # Public: The name of the adapter.
+      attr_reader :name
+
+      # Public
+      def initialize(adapter, cache, expires_in: nil)
+        @adapter = adapter
+        @name = :cache_store
+        @cache = cache
+        @write_options = {}
+        @write_options.merge!(expires_in: expires_in) if expires_in
```

Comment:
ActiveSupport::Cache will use ttl set on the `@cache` getting passed into this initializer, but allows you to set a ttl for a given key when using fetch/write.  Doing it this way we won't clobber the already set ttl, but still allow users to use a different ttl for flipper queries.

Its also possible that users won't even want to use a different ttl here and will always want to use whatever they're cache store is using when they initialize it?